src/Helpers/TagsHelper.cs

```diff
@@ -36,8 +41,8 @@ public static ContentReference GetContentReference(Guid contentGuid)
 
         public static bool IsTagProperty(PropertyDefinition propertyDefinition)
         {
-            return propertyDefinition != null
-                   && propertyDefinition.TemplateHint == "Tags";
+            return propertyDefinition != null && 
```

Comment:
`&&` is more visible when is on the new line as before.

Reply:
My eyes hurt, but np.